Tests/UITests/TestApp/LLMOpenAIRealtime/AudioViewModel.swift

```diff
@@ -0,0 +1,95 @@
+//
+// This source file is part of the Stanford Spezi open source project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SpeziLLMOpenAIRealtime
+import SwiftUI
+
+@Observable
+@MainActor
+final class AudioViewModel {
```

Comment:
This type might be very helpful to be included in the main target or a target that can be imported on top of the main target. We should ideally polish this a bit and find a nice mechanism to provide a compact public API for this that can be used on Apple platforms (and would need to be conditionally excluded on Linux).